docs/didkit/install.md

```diff
@@ -53,19 +80,23 @@ $ docker run ghcr.io/spruceid/didkit-cli:latest --help
 $ docker run --init -p 8080 ghcr.io/spruceid/didkit-http:latest --port 8080
 ```
 
-> Note: You can pass JWKs either by sharing a volume with `docker run --volume`, or by passing the JWK directly with `docker run -e JWK=$MY_JWK` or `docker run didkit-http --jwk $MY_JWK`.
+> Note: You can pass JWKs either by sharing a volume with `docker run --volume`,
+> or by passing the JWK directly with `docker run -e JWK=$MY_JWK` or `docker run
+> didkit-http --jwk $MY_JWK`.
 
 ### Build Images
 
-The Dockerfiles rely on having [ssi][] in the root of `didkit` (a symbolic link will not work, unfortunately).
+The Dockerfiles rely on having [ssi][] in the root of `didkit` (a symbolic link
```

Comment:
It's probably not true anymore but I'd need to check.

Reply:
worth checking, i'd say?

Reply:
Ok, they still do. I don't think they need to but it's fine (it ensures the image has the latest ssi).